.dockerignore

```diff
@@ -0,0 +1,5 @@
+Dockerfile
+.dockerignore
```

Comment:
why do we ignore the Dockerfile?

Reply:
It just means, that we will not copy the Dockerfile into the Container.